recipe/0514-ml-usage-tags/manifest.json

```diff
@@ -0,0 +1,78 @@
+{
+  "@context": "http://iiif.io/api/presentation/3/context.json",
+  "id": "{{ id.url }}",
+  "type": "Manifest",
+  "label": {
+    "en": [
+      "Picture of Göttingen taken during the 2019 IIIF Conference"
+    ]
+  },
+  "summary": {
+    "en": [
+      "<p>Picture taken by the <a href=\"https://github.com/glenrobson\">IIIF Technical Coordinator</a></p>"
+    ]
+  },
+  "rights": "http://creativecommons.org/licenses/by-sa/3.0/",
+  "requiredStatement": [
```

Comment:
I don't think requiredStatement can be an array I think it needs to be a JSON object. So to get it to validate if you can remove the [] in required statement it should get past the validation and start deploying the preview.

Reply:
Ah got it, sorry for the extra [], removed

Reply:
Sorry I'm being dim. Its not valid JSON now without the `[]` and looking at the presentation API:

https://iiif.io/api/presentation/3.0/#requiredstatement

I think its only possible to have one required statement. For this example do you want to remove the attribution one?

Reply:
Ok, thank you, just sent through another updated version with the attribution statement as a second value, hope that’s alright.

Reply:
This exemplifies the problem with the approach. Now you have two values with one label, but the label only applies to the first. This should be called out in the recipe description, and that this is a field intended for humans to read rather than machines to process. Even the best intentioned machine agent won't know what to do with this... at which point, just merge the two parts into one statement.

Reply:
Thanks for the feedback @azaroth42.

My reading of the specs is indeed that the `requiredStatement` property is really only for human readable and displayable statements (recipe states '[for humans](https://github.com/IIIF/cookbook-recipes/blob/507c14055a5cb82d5fa03ba115460a2ce48565ad/recipe/0514-ml-usage-tags/index.md?plain=1#L24)', but could be more verbose about this in the general description text as well). I'm thinking of this initial recipe as complementary to and a first step towards the more (intentionally) machinable `rights` (have a note about the WIP here:  
https://github.com/IIIF/cookbook-recipes/blob/507c14055a5cb82d5fa03ba115460a2ce48565ad/recipe/0514-ml-usage-tags/index.md?plain=1#L61

(FWIW, also have a [brief note about the actionability in general](https://github.com/IIIF/cookbook-recipes/blob/507c14055a5cb82d5fa03ba115460a2ce48565ad/recipe/0514-ml-usage-tags/index.md?plain=1#L20))

This is a bit out of scope for this particular recipe, but It might be worth having a larger discussion about `requiredStatement` only permitting a single object. I can see the use case for multiple objects/statements (regardless of these potential tags), such as repositories that often have both standardized and local rights statements, where it would helpful to users to see the text for the standardized statement in full alongside the more local statements. Or where multiple language labels and paired value statements could be useful.